Dockerfile

```diff
@@ -42,6 +42,12 @@ RUN wget -O- https://github.com/mozilla/sops/releases/download/v3.7.1/sops-v3.7.
   echo "185348fd77fc160d5bdf3cd20ecbc796163504fd3df196d7cb29000773657b74  /usr/local/bin/sops" | sha256sum -c - && \
   chmod +x /usr/local/bin/sops && sops -v
 
+# age (encryption for sops)
+RUN wget -O- 'https://dl.filippo.io/age/v1.0.0?for=linux/amd64' > age.tar && \
```

Comment:
Note this adds a dependency on age to our build process (actually and as-written it adds a dependency on the author's server, which just issues a redirect to github- feel free to tell me i should paste that redirect in here so at least we only depend on github)